scripts/collect-latest.py

```diff
@@ -4,8 +4,8 @@
 import os
 import sys
 
-with open("scripts/sbom-locations.json") as data:
-    locations = json.load(data)
+with open("scripts/sboms-from-maven-central.json") as data:
```

Comment:
the idea was that this file could also specify where/how to fetch SBOMs for non-Maven Central artifacts - but we didn't have any generalized cases for that yet. I guess this is fine, we can evolve it further later when we do have such generalized cases.